ui/src/app/pages/workspace/workspace-share.spec.tsx

```diff
@@ -170,10 +169,8 @@ describe('WorkspaceShare', () => {
   });
 
   it('adds user correctly', async () => {
-    const clock = Lolex.install({ shouldAdvanceTime: true });
```

Comment:
I'm curious what these changes are about... I was actually planning to make an alternate change soon: #6913

Reply:
I forget the error message. Something like "cannot replace 'performance' property."